src/store/basket.ts

```diff
@@ -0,0 +1,37 @@
+import { AnyAction } from 'redux';
+import { Product, Basket } from '../interfaces';
+import { removeDuplicates, deleteItemFromBasket } from '../helpers/removeDuplicates';
+
+const SET_BASKET = 'SET_BASKET';
+const REMOVE__ITEM = 'REMOVE__ITEM';
+const DELETE__ITEM = 'DELETE__ITEM';
+
+export const setBasket = (basketItem: Product) => ({ type: SET_BASKET, payload: basketItem });
+export const removeItem = (basketItem: Product) => ({ type: REMOVE__ITEM, payload: basketItem });
+export const deleteItem = (id: string) => ({ type: DELETE__ITEM, payload: id });
+
+let initState: Basket[] = [];
```

Comment:
```suggestion
let initialState: Basket[] = [];
```
на самом деле разница между init и initial большая